src/code/z_jpeg.c

```diff
@@ -356,8 +358,8 @@ s32 Jpeg_Decode(void* data, void* zbuffer, void* work, u32 workSize) {
     curTime = osGetTime();
     diff = curTime - time;
     time = curTime;
-    // "Unfold & draw"
-    PRINTF("*** 展開 & 描画 time = %6.3f ms ***\n", OS_CYCLES_TO_USEC(diff) / 1000.0f);
+    PRINTF(T("*** 展開 & 描画 time = %6.3f ms ***\n", "*** Unfold & draw time = %6.3f ms ***\n"),
+           OS_CYCLES_TO_USEC(diff) / 1000.0f);
```

Comment:
I think "unfold" is very clearly "decode" based on context, and they even use "decode" in english a couple prints above. but wont push for this change now if you dont want to

Reply:
Yeah I understand that is more the point of what youre doing. I do feel compelled to still comment my thoughts on the translations, cause its not often I have a reason to sit down and read all of them. You can choose to act on them/not act on them as you please or make note of TODOs anywhere as it fits